changeset:   1793:c3d82278a098
user:        tytso@mit.edu
date:        Tue Sep  6 06:42:49 2005 -0400
summary:     Add support for detecting software suspend partitions to blkid

diff -r 6949f7a75122 -r c3d82278a098 lib/blkid/probe.c
--- a/lib/blkid/probe.c	Tue Sep  6 10:26:45 2005
+++ b/lib/blkid/probe.c	Tue Sep  6 10:42:49 2005
@@ -529,14 +529,24 @@
   { "sysv",	 0,  0x3f8,  4, "\020~\030\375",	0 },
   { "swap",	 0,  0xff6, 10, "SWAP-SPACE",		probe_swap0 },
   { "swap",	 0,  0xff6, 10, "SWAPSPACE2",		probe_swap1 },
+  { "swsuspend", 0,  0xff6,  9, "S1SUSPEND",		probe_swap1 },
+  { "swsuspend", 0,  0xff6,  9, "S2SUSPEND",		probe_swap1 },
   { "swap",	 0, 0x1ff6, 10, "SWAP-SPACE",		probe_swap0 },
   { "swap",	 0, 0x1ff6, 10, "SWAPSPACE2",		probe_swap1 },
+  { "swsuspend", 0, 0x1ff6,  9, "S1SUSPEND",		probe_swap1 },
+  { "swsuspend", 0, 0x1ff6,  9, "S2SUSPEND",		probe_swap1 },
   { "swap",	 0, 0x3ff6, 10, "SWAP-SPACE",		probe_swap0 },
   { "swap",	 0, 0x3ff6, 10, "SWAPSPACE2",		probe_swap1 },
+  { "swsuspend", 0, 0x3ff6,  9, "S1SUSPEND",		probe_swap1 },
+  { "swsuspend", 0, 0x3ff6,  9, "S2SUSPEND",		probe_swap1 },
   { "swap",	 0, 0x7ff6, 10, "SWAP-SPACE",		probe_swap0 },
   { "swap",	 0, 0x7ff6, 10, "SWAPSPACE2",		probe_swap1 },
+  { "swsuspend", 0, 0x7ff6,  9, "S1SUSPEND",		probe_swap1 },
+  { "swsuspend", 0, 0x7ff6,  9, "S2SUSPEND",		probe_swap1 },
   { "swap",	 0, 0xfff6, 10, "SWAP-SPACE",		probe_swap0 },
   { "swap",	 0, 0xfff6, 10, "SWAPSPACE2",		probe_swap1 },
+  { "swsuspend", 0, 0xfff6,  9, "S1SUSPEND",		probe_swap1 },
+  { "swsuspend", 0, 0xfff6,  9, "S2SUSPEND",		probe_swap1 },
   { "ocfs",	 0,	 8,  9,	"OracleCFS",		probe_ocfs },
   { "ocfs2",	 1,	 0,  6,	"OCFSV2",		probe_ocfs2 },
   { "ocfs2",	 2,	 0,  6,	"OCFSV2",		probe_ocfs2 },
